src/System.Text.Json/src/System/Text/Json/ThrowHelper.cs

```diff
@@ -592,6 +592,11 @@ public static FormatException GetFormatException(DataType dateType)
             ex.Source = ExceptionSourceValueToRethrowAsJsonException;
             return ex;
         }
+
+        public static InvalidOperationException GetInvalidOperationException_ExpectedChar(JsonTokenType tokenType)
```

Comment:
nit: why is this all the way down here and not with the other `*_ExpectedX` methods starting at L186

Reply:
That's what we have checked-in master. We can do this code clean up there if its makes sense to do so, but has no meaningful impact.